Server/BackendClasses/Encryption.cs

```diff
@@ -5,9 +5,37 @@
 using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web;
+using System.Web.Security;
 
 namespace LauncherServerClasses
 {
+    public static class MachineKeyEncryption
```

Comment:
This class is added to provide a different encryption method for hiding the secret key, locally, on the client machine. The "Protect" and "UnProtect" methods are used for encrypting and decrypting the secret key using the machine key, instead of using RSA.